pkg/baseline/packageIndex.json

```diff
@@ -9,8 +9,9 @@
       "BaselineVersion": "4.4.0",
       "InboxOn": {},
       "AssemblyVersionInPackageVersion": {
-        "4.1.1.0": "4.4.0",
-        "4.1.2.0": "4.4.0"
+        "4.1.1.0": "4.5.0",
```

Comment:
> if an assembly is already released, how can I decide not to ship it in a previous release?

You can't but the index could have been wrong.  For example you were building a new version of that assembly but then decided to harvest it but the index wasn't updated.